CIP-0116/cardano-babbage.json

```diff
@@ -0,0 +1,1952 @@
+      "required": [
+        "url",
+        "hash"
+      ]
+    },
+    "PoolPubKeyHash": {
+      "title": "PoolPubKeyHash",
+      "type": "string",
+      "format": "bech32",
+      "pattern": "^(pool1)[02-9ac-hj-np-z]*$"
+    },
+    "PoolParams": {
+      "title": "PoolParams",
+      "type": "object",
+      "properties": {
+        "operator": {
+          "$ref": "cardano-babbage.json#/definitions/PoolPubKeyHash"
+        },
+        "vrf_keyhash": {
+          "$ref": "cardano-babbage.json#/definitions/VRFKeyHash"
+        },
+        "pledge": {
+          "$ref": "cardano-babbage.json#/definitions/UInt64"
+        },
+        "cost": {
+          "$ref": "cardano-babbage.json#/definitions/UInt64"
+        },
+        "margin": {
+          "$ref": "cardano-babbage.json#/definitions/UnitInterval"
+        },
+        "reward_account": {
+          "$ref": "cardano-babbage.json#/definitions/RewardAddress"
+        },
+        "pool_owners": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/Ed25519KeyHash"
+          }
+        },
+        "relays": {
+          "type": "array",
+          "items": {
+            "$ref": "cardano-babbage.json#/definitions/Relay"
+          }
+        },
+        "pool_metadata": {
+          "$ref": "cardano-babbage.json#/definitions/PoolMetadata"
+        }
+      },
+      "required": [
+        "cost",
+        "margin",
+        "operator",
+        "pledge",
+        "pool_owners",
+        "relays",
+        "reward_account",
+        "vrf_keyhash"
+      ],
+      "additionalProperties": false
+    },
+    "GenesisHash": {
+      "title": "GenesisHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{56}$"
+    },
+    "GenesisDelegateHash": {
+      "title": "GenesisDelegateHash",
+      "type": "string",
+      "format": "hex",
+      "pattern": "^[0-9a-f]{56}$"
+    },
+    "MIRPot": {
+      "title": "MIRPot",
+      "enum": [
+        "reserves",
+        "treasury"
+      ]
+    },
+    "MoveInstantaneousRewards": {
+      "title": "MoveInstantaneousRewards",
+      "type": "object",
+      "discriminator": {
+        "propertyName": "tag"
+      },
+      "oneOf": [
+        {
+          "type": "object",
+          "title": "Move Instantaneous Rewards to stake credentials",
+          "properties": {
+            "tag": {
+              "enum": [
+                "to_stake_creds"
+              ]
+            },
+            "pot": {
+              "$ref": "cardano-babbage.json#/definitions/MIRPot"
+            },
+            "rewards": {
+              "title": "MIRToStakeCredentials",
+              "description": "Distribution of rewards",
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "key": {
+                    "$ref": "cardano-babbage.json#/definitions/Credential"
```

Comment:
shouldnt this be stake account/ reward address?

Reply:
A reward address is more than a credential because it includes a network tag.

```cddl
move_instantaneous_reward = [ 0 / 1, { * stake_credential => delta_coin } / coin ]

stake_credential =
  [  0, addr_keyhash
  // 1, scripthash
  ]
```

Reply:
Good catch